package.json

```diff
@@ -22,6 +22,9 @@
     "mysql": "^2.17.1"
   },
   "devDependencies": {
-    "cucumber": "^5.1.0"
+    "cucumber": "^5.1.0",
+    "request": "^2.88.0",
+    "chai": "*",
+	"mocha": "*"
```

Comment:
I think we should do ```npm install <package-name> --save-dev``` and then push the```package.json``` and ```package-lock.json``` generated from that action.

That way we have a version number for the ```chai``` and ```mocha``` dependencies. That way the GitHub dependabot can help keep those packages up to date by issuing pull-requests for outdated packages.